feature/platform/tests/telemetry_inventory_test/README.md

```diff
@@ -21,28 +21,53 @@ card, power supply, disk, flash, NPU, transceiver, fabric card), validate:
 
 ## Telemetry Parameter coverage
 
-*   /components/component[name=<heatsink-temperature-sensor>]/state/temperature/instant
-*   /components/component/storage
-*   TODO: /components/component/software-module
-*   TODO: /components/component/software-module/state/module-type
-*   /components/component/state/description
-*   /components/component/state/firmware-version
-*   /components/component/state/hardware-version
-*   /components/component/state/id
-*   /components/component/state/mfg-date
-*   /components/component/state/mfg-name
-*   /components/component/state/name
-*   /components/component/state/oper-status
-*   /components/component/state/parent
-*   /components/component/state/part-no
-*   /components/component/state/serial-no
-*   /components/component/state/software-version
-*   /components/component/state/type
-*   /components/component/state/temperature/alarm-status
-*   /components/component/state/temperature/instant
-*   /components/component/state/temperature/max
-*   /components/component/state/temperature/max-time
-*   /components/component/integrated-circuit/backplane-facing-capacity/state/available-pct
-*   /components/component/integrated-circuit/backplane-facing-capacity/state/consumed-capacity
-*   /components/component/integrated-circuit/backplane-facing-capacity/state/total
-*   /components/component/integrated-circuit/backplane-facing-capacity/state/total-operational-capacity
+```yaml
+paths:
+    TODO: /components/component/storage
+    TODO: /components/component/software-module
+    TODO: /components/component/software-module/state/module-type
+    /components/component/state/description
+       patform_type: ["CHASSIS", "CONTROLLER_CARD", "FABRIC", "FAN", "LINECARD", "POWER_SUPPLY"]
+    /components/component/state/firmware-version
+       patform_type: ["TRANSCEIVER"]
+    /components/component/state/hardware-version
+       patform_type: ["CHASSIS", "CONTROLLER_CARD", "FABRIC", "LINECARD", "POWER_SUPPLY", "TRANSCEIVER"]
+    /components/component/state/id
+       patform_type: ["CONTROLLER_CARD", "FABRIC", "FAN", "INTEGRATED_CIRCUIT", "LINECARD", "POWER_SUPPLY", "SENSOR"]
+   /components/component/state/mfg-date
+       patform_type: []
```

Comment:
@dplore I don't believe that the mfg-date and software-version paths are validated for any type of component (`mfgDateValidation` and `swVerValidation` are always false).

I don't want to not include `platform_type` for the risk of it being misunderstood that these paths are validated for all types, which is why I have included an empty list here. Alternatively, should these two paths just be removed from this list?

Reply:
You may comment them out with a "TODO" comment.